### ISSUE TITLE
Marking ports

### ISSUE THREAD

**Issue by lunar-crater-ex** [citadel-station/sector-xiv#9]
tl;dr SS14's existing set of markings is fairly small, porting markings from other servers would be nice for character customization.

## But how?
Look into how other markings, [like the cat ears](https://github.com/Citadel-Station-13/space-station-14/blob/master/Resources/Prototypes/Species/human.yml), are set up. For porting assets, you'll want to use https://github.com/space-wizards/RSIEdit (Downloads are provided, no need to compile it) to create new RSIs for the ported markings.
**Yes, I will be requiring you to include proper attribution in RSIs, will provide assistance with this if needed.**
